src/main/kotlin/com/cognifide/gradle/aem/base/vlt/VltRunner.kt

```diff
@@ -7,6 +7,7 @@ import com.cognifide.gradle.aem.internal.file.FileOperations
 import com.cognifide.gradle.aem.pkg.PackagePlugin
```

Comment:
finally, do not touch it, reuse

Reply:
That's why the reusable part is extracted to CheckoutConfig